### PR TITLE
Zendoo integration all

### PR DESCRIPTION
**Pull request by oaleksandrov-ip**

### CODE REVIEW

File: qa/readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        ```
+
+**Template configuration files**
+
+Template configuration files are located in directory resources. 
+
+File template.conf is template for tests of SC node(s) is connected to MC node(s).
+
+File template_predefined_genesis.conf is template for test of SC node(s) in standalone mode(without connection to MC node).
+
```
Comment (i-Alex) [inline qa/readme.md:33]: We need to add info how to set MC cli/zend and SC boot tool and simple app (or any other app)

File: sdk/pom.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>com.horizen</groupId>
             <artifactId>zendoo-sc-cryptolib</artifactId>
-            <version>0.2.1-SNAPSHOT</version>
+            <version>0.2-SNAPSHOT</version>
```
Comment (i-Alex) [inline sdk/pom.xml:47]: is it correct? Why not 0.2.6 generated yesterday?

File: sdk/pom.xml [hunk 3]
```diff
@@ -311,6 +305,7 @@
                     <include>schnorr_sk00_hex</include>
                     <include>sample_proving_key_7_keys_with_threshold_5</include>
                     <include>sample_vk_7_keys_with_threshold_5</include>
+					<include>pippo_vk</include>
```
Comment (i-Alex) [inline sdk/pom.xml:308]: can be removed.
And moreover, let's have a regression test for proof verification.

File: sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java [hunk 4]
```diff
@@ -63,8 +58,8 @@ public ForgerBox getBox() {
         );
     }
 
-    public byte[] getBackwardTransferPoseidonRootHash() {
-        return new byte[0];
+    public byte[] genSysConstant() {
```
Comment (i-Alex) [inline sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java:61]: Probably,  `genSysConstant` -> `getSysConstant`

File: sdk/pom.xml (hunk 2 above)
Comment (oaleksandrov-ip) [reply sdk/pom.xml:47]: we need to define it, current version in development branch in zensoo-sc-cryptolib is 0.2-SNAPSHOT

File: sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java (hunk 4 above)
Comment (oaleksandrov-ip) [reply sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java:61]: we use the same name `genSysConstant` everywhere, so it's not missprint

Comment (i-Alex) [reply sdk/src/main/java/com/horizen/transaction/mainchain/SidechainCreation.java:61]: ok, i got it. Thought that `gen` is `generate`, but it's actually like get generated sys constant

**Pull request merged**